auth-source-pass.el

```diff
@@ -38,6 +38,16 @@
 (require 'auth-source)
 (require 'url-parse)
 
+(defgroup auth-source-pass nil
+  "password-store integration within auth-source."
+  :prefix "auth-source-pass-"
+  :group 'auth-source)
+
+(defcustom auth-source-pass-path "~/.password-store"
+  "Path to the password-store folder."
+  :type 'directory
+  :group 'auth-source-pass)
```

Comment:
please remove the `:group` line as it is optional (by default, the group is the one defined in the same file).